jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -161,4 +165,11 @@ boolean getAllowInsecureRegistries() {
     }
     return allowInsecureRegistries.get();
   }
+
+  @Input
+  @Optional
+  public String getContainerizingMode() {
+    String property = System.getProperty(PropertyNames.CONTAINERIZING_MODE);
```

Comment:
do we allow system properties for all values?

Reply:
Yeah, we've been doing this for config parameters we offer.

Reply:
It allows applying jib to a project without any modification.